src/PactNet.Native/Scenarios.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using PactNet.Models;
+
+namespace PactNet.Native
+{
+    /// <summary>
+    /// Defines the scenarios static list for messaging support
+    /// </summary>
+    public class Scenarios
```

Comment:
Is this class supposed to be public? Aren't users supposed to use the builders in the main `PactNet` project to define these?

Reply:
No, the Scenarios class is the inmemory storage for the scenarios and need to be public